vip-filesystem.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Plugin Name:       VIP Filesystem Plugin
+ * Description:       Provides a custom stream wrapper for handling file uploads to VIP/A8C Files API.
+ * Version:           1.0.0
+ * Author:            Automattic
+ * Author URI:        http://automattic.com/
+ * License:           GPL-2.0+
+ * License URI:       http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+// If this file is called directly, abort.
+if ( ! defined( 'WPINC' ) ) {
+	die;
+}
+/**
+ * Currently plugin version.
+ * Start at version 1.0.0 and use SemVer - https://semver.org
+ */
+define( 'VIP_FILESYSTEM_VERSION', '1.0.0' );
+
+/**
+ * The core plugin class
+ */
+require __DIR__ . '/vip-filesystem/class.vip-filesystem.php';
```

Comment:
Should we put all these files into `/files/` alongside all the other files related to the Files Service? (If so, we'll want to update the Namespace, even though we're not super consistent with naming :))